src/System.Private.CoreLib/shared/System/IO/Path.cs

```diff
@@ -824,5 +824,33 @@ internal static StringComparison StringComparison
                     StringComparison.OrdinalIgnoreCase;
             }
         }
+
+        /// <summary>
+        /// Trims any trailing separators beyond the root of the path.
```

Comment:
"Trims any trailing separators..." makes it sound like that it trims all of them if there are multiple. (Correct me if I am wrong. English is not native language.)

Should this rather say "Trims one trailing directory separator beyond the root of the path" ?

Reply:
I had same doubt, but my doubts on my english are greater :smile:

Reply:
I had same doubt, but my doubts on my english are greater :smile:

Reply:
/cc @JeremyKuhne

Reply:
@jkotas suggestion is good. Do you need anything else or is this then good to merge? Sorry it took so long.

Reply:
No I'll fix comment and is ready to go to me